app/javascript/mastodon/locales/ja.json

```diff
@@ -102,23 +102,23 @@
   "home.column_settings.show_reblogs": "ブースト表示",
   "home.column_settings.show_replies": "返信表示",
   "home.settings": "カラム設定",
-  "keyboard_shortcuts.back": "to navigate back",
-  "keyboard_shortcuts.boost": "to boost",
-  "keyboard_shortcuts.column": "to focus a status in one of the columns",
-  "keyboard_shortcuts.compose": "to focus the compose textarea",
-  "keyboard_shortcuts.description": "Description",
-  "keyboard_shortcuts.down": "to move down in the list",
-  "keyboard_shortcuts.enter": "to open status",
-  "keyboard_shortcuts.favourite": "to favourite",
-  "keyboard_shortcuts.heading": "Keyboard Shortcuts",
-  "keyboard_shortcuts.hotkey": "Hotkey",
-  "keyboard_shortcuts.legend": "to display this legend",
-  "keyboard_shortcuts.mention": "to mention author",
-  "keyboard_shortcuts.reply": "to reply",
-  "keyboard_shortcuts.search": "to focus search",
-  "keyboard_shortcuts.toot": "to start a brand new toot",
-  "keyboard_shortcuts.unfocus": "to un-focus compose textarea/search",
-  "keyboard_shortcuts.up": "to move up in the list",
+  "keyboard_shortcuts.back": "戻る",
+  "keyboard_shortcuts.boost": "ブースト",
+  "keyboard_shortcuts.column": "左からn番目のカラム内最新トゥートに移動",
+  "keyboard_shortcuts.compose": "トゥート入力欄に移動",
+  "keyboard_shortcuts.description": "説明",
+  "keyboard_shortcuts.down": "カラム内一つ下に移動",
+  "keyboard_shortcuts.enter": "トゥートの詳細を表示",
+  "keyboard_shortcuts.favourite": "お気に入り",
+  "keyboard_shortcuts.heading": "キーボードショートカット一覧",
+  "keyboard_shortcuts.hotkey": "ホットキー",
+  "keyboard_shortcuts.legend": "この一覧を表示",
+  "keyboard_shortcuts.mention": "メンション",
```

Comment:
Just a reminder, I think in Japanese [`mention` is also translated as `返信`](https://github.com/mayaeh/mastodon/blob/d1a58bf9209a6a59617679c79f849bf47a122517/app/javascript/mastodon/locales/ja.json#L207) at the moment. I'm not a native speaker though.

Reply:
Thank you.

I made separate translations, because mention and reply have slightly different functions.

I translated "Reply" as "返信".
"返信" means reply to something, and there is a target to return.
Reply function is a reply to TOOT, and it isn't displayed on the local TL etc.
Also reply function is displayed as a series of conversations.

And I translated "Mention" as "メンション".
Mention function is a reply to a person, and it is also displayed in the local TL etc.
I think that Mention function of SNS service is often used mainly when contact someone.

Also "Mention" is often translated as "言及" in Japanese.
"言及" is a hard expression and is not used much in everyday conversation.

I think that other SNS service also translates "Mention" as "メンション" and uses it.

So I translated "Mention" into "メンション".

Reply:
I know, I'm just talking about unification here, in order to avoid confusions :)